from unittest import TestCase


class Test(TestCase):
    def test_has_sustained(self):
        self.fail()
